### DISCUSSION TITLE
Getting imported unit wrong for a recipe site

### DISCUSSION THREAD

**Discussion by scottb721** [mealie-recipes/mealie#3739] [Q&A] [answered]
I've tried two recipes from [Taste.com.au](https://www.taste.com.au/) and both have had Cup measurements replaced with ml.
eg 1/4 cup becomes 62.5ml 
https://www.taste.com.au/recipes/easy-seafood-laksa/7a86bc26-e04f-460d-ad82-f8531733b9a0

Is this something I can prevent from occurring in the settings or do I need to manually correct the recipe ?

Thanks

**Comment by Mraedis** [comment, marked as answer]
The recipe parser grabs the schema.org JSON on the page, which lists the ingredients in metric. The taste site converts this on the front-end but there's nothing you can do to import it in imperial I'm afraid.

**Reply by scottb721** [reply]
Thanks for that. So the Taste site is converting the ml value in their backend data into cups for the webpage ?

**Reply by Mraedis** [reply]
Front-end, but yes. You can see it if you look at the page source and look for, for instance, 62.5 ml:
![image](https://github.com/mealie-recipes/mealie/assets/3253734/beec5ae6-c9ce-43b0-ab1a-03c144ad36a0)

If you're interested in the technical aspects of this, check out https://schema.org/Recipe
and https://github.com/hhursev/recipe-scrapers (which is what mealie uses to scrape recipes).

**Reply by scottb721** [reply]
Thanks for the information. Didn't know that was even a thing 😂
